marquez.dev.yml

```diff
@@ -2,6 +2,7 @@ server:
   applicationConnectors:
   - type: http
     port: 5000
+    httpCompliance: RFC2616_LEGACY
```

Comment:
Would it be better to use `RFC7230_LEGACY` instead since `RFC7230` is the [default for Dropwizard](https://www.dropwizard.io/en/latest/manual/configuration.html#http)? With my basic testing this also allows forward slashes.